training-react/login/src/app/components/App/components/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Login from './layout';
+
+const LoginContainer = () => <Login />;
```

Comment:
For the functional components you shouldn't use arrow functions, because is a bad practice. Check this [article](https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless)